thymeleaf-3.0.9.RELEASE/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <!--
+
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+  -->
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.servicemix.bundles</groupId>
+    <artifactId>bundles-pom</artifactId>
+    <version>13</version>
+    <relativePath>../bundles-pom/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>org.apache.servicemix.bundles.thymeleaf</artifactId>
+  <version>3.0.9.RELEASE_1-SNAPSHOT</version>
+  <packaging>bundle</packaging>
+  <name>Apache ServiceMix :: Bundles :: ${pkgArtifactId}</name>
+  <description>This OSGi bundles wraps ${pkgArtifactId} ${pkgVersion} jar file.</description>
+
+  <scm>
+    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/servicemix-bundles.git</connection>
+    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/servicemix-bundles.git</developerConnection>
+    <url>https://git-wip-us.apache.org/repos/asf?p=servicemix-bundles.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <pkgGroupId>org.thymeleaf</pkgGroupId>
+    <pkgArtifactId>thymeleaf</pkgArtifactId>
+    <pkgVersion>3.0.9.RELEASE</pkgVersion>
+    <servicemix.osgi.export>${servicemix.osgi.export.pkg}</servicemix.osgi.export>
+    <servicemix.osgi.export.pkg>
```

Comment:
Only `servicemix.osgi.export.pkg` property is required. Please remove `servicemix.osgi.export`.

Reply:
In that case, don't provide the pkg property. See other bundle for example.